static/popups/Upload.jsx

```diff
@@ -76,28 +76,18 @@ class ReactUpload extends React.Component {
   }
 
   onDrop(files) {
+    var fd= new FormData();
```

Comment:
These updates are awesome!
```suggestion
    const fd = new FormData();
```